AdminCore.WebApp/Controllers/TeamController.cs

```diff
@@ -46,6 +47,20 @@ public IActionResult GetTeamById(int id)
       return StatusCode((int)HttpStatusCode.InternalServerError, $"No team found with an ID of { id.ToString() }");
     }
 
+    [HttpGet("getTeamsByProjectId/{projectId}")]
+    [ProducesResponseType(typeof(IList<TeamViewModel>), (int)HttpStatusCode.OK)]
+    [ProducesResponseType(StatusCodes.Status204NoContent)]
+    public IActionResult GetTeamsByProjectById(int projectId)
+    {
+      var teamList = _teamService.GetByProjectId(projectId);
+      if (teamList == null || !teamList.Any())
```

Comment:
Hmmm... what if projectId is null, you end up with `teamList` null or empty, then you call `ToString()` on the projectId and 💥. 

Also if projectId is empty you'll end up with crappy log line.

Reply:
consider not querying for `teamList()` if projectId is useless

Reply:
I would like to see these scenarios tested

Reply:
ProjectId is an int, it isn't a nullable type so it won't run into the situation that it is null. If you try and hit that endpoint with an invalid int then the middleware will handle that for you and return a 400 bad request I think.

Reply:
sounds like something c# specific, can you ensure these scenarios are handled correctly and are tested

Reply:
interesting, this sounds like it's c# specific